doc/script_commands.txt

```diff
@@ -6910,24 +6912,6 @@ command, simply use 1 for type. Any other number won't be recognized.
 
 ---------------------------------------
 
-*strmobinfo(<type>,<monster id>);
```

Comment:
Sadly we have to restore it for the time being.
See comments below.
Please add a deprecation note here.

Reply:
imo deleting the doc as crazy did is just best

Reply:
I didn't add back the documentation since It's better to keep it out of the docs